source/client/world/ClientPlayer.cpp

```diff
@@ -72,54 +72,89 @@ void ClientPlayer::updateCamera() {
 	m_camera.setUpVector(gk::Vector3f{sh * sr - ch * sv * cr, -ch * sr - sh * sv * cr, cv * cr});
 }
 
-void ClientPlayer::move(float direction) {
-	direction += m_viewAngleH;
+void ClientPlayer::move(const ClientWorld &world, float direction) {
+	ClientChunk *chunk = (ClientChunk *)world.getChunkAtBlockPos(m_x, m_y, m_z);
+	if (chunk && chunk->isInitialized()) {
+		direction += m_viewAngleH;
```

Comment:
Why did you add this check here?

Reply:
I was mimicking what the gravity code uses IIRC

Reply:
the function needed that check to block player movement if he was in an unloaded chunks, that's why it wasn't in other functions